examples/photoCalTask.py

```diff
@@ -75,19 +70,25 @@ def run():
     exposure, srcCat = loadData()
     schema = srcCat.getSchema()
     #
+    # Create the reference catalog loader
+    #
+    refCatDir = os.path.join(lsst.utils.getPackageDir('meas_astrom'), 'tests', 'data', 'sdssrefcat')
```

Comment:
That data appears in both packages, and is identical in both. I'm not thrilled about this duplication, but in general we prefer test data in packages to be self-contained, so that modifying test data in one package won't mysteriously break tests in other packages. And indeed both packages had duplicate a.net versions of this data (`tests/astrometry_net_data/photocal`). I was able to delete one of those, and the other will move to the new a.net wrapper package as part of separating moving the a.net wrapper code into its own package.